x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_table_panel_section.tsx

```diff
@@ -159,11 +159,15 @@ export const VulnerabilityTablePanelSection = () => {
               defaultMessage: 'CVSS',
             }
           ),
-          render: (cvss: { score: number; version: string }) => (
-            <EuiLink onClick={() => onCellClick({ 'vulnerability.score.base': cvss.score })}>
-              <CVSScoreBadge version={cvss.version} score={cvss.score} />
-            </EuiLink>
-          ),
+          render: (cvss: { score: number; version: string }) => {
```

Comment:
```suggestion
          render: (cvss: Vulnerability["score"]) => {
```

Reply:
@JordanSh thanks for the suggestion! the `Vulnerability["score"]` is not the type we can use here, but I changed the code to use the correct type (as we are passing formatted data to the table)